pkgs/development/tools/misc/universal-ctags/default.nix

```diff
@@ -12,7 +12,8 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pythonPackages.docutils ];
-  buildInputs = [ autoreconfHook pkgconfig ];
+  buildInputs = [ autoreconfHook pkgconfig ] ++
```

Comment:
`autoreconfHook` and `pkgconfig` should be native build inputs. I fixed this on staging so doing it here might just help us avoid a merge conflict.